src/gsumm.c

```diff
@@ -1013,6 +1017,16 @@ SEXP glast(SEXP x) {
       if (isunsorted) k = oo[k]-1;
       k = (irowslen == -1) ? k : irows[k]-1;
       ians[i] = ix[k];
+      if (narm){
+        j = 0;
+        while(j < grpsize[i]){
```

Comment:
what happens when the group is all-`NA`?

Reply:
Good question. Also raised this question in #4239 

> Whats happens in other `gfunctions` for those groups?
> 
> * `gsum` returns 0
> * `gmedian`, `gvar` and `gsd` return NA
> * `gmean` returns NaN
> * `gprod` returns 1
> * `gmax`/`gmin` return Inf

Personally I find `NA` intuitive since I would not want groups to disappear by grouping functions.